src/main/java/ojosama/talkak/comment/service/CommentService.java

```diff
@@ -45,7 +46,8 @@ public List<CommentResponse> getCommentsByVideoId(long videoId) {
             .toList();
     }
 
-    public CommentResponse updateComment(Long commentId, Long memberId, CommentRequest commentRequest) {
+    public CommentResponse updateComment(Long commentId, Long memberId,
+        CommentRequest commentRequest) {
         Comment comment = commentRepository.findById(commentId)
             .orElseThrow(() -> TalKakException.of(CommentError.INVALID_COMMENT_ID));
         if (!Objects.equals(comment.getMember().getId(), memberId)) {
```

Comment:
```suggestion
        if (comment.isCommenter(memberId)) {
```
comment 의 member 를 조회하고 id 를 가져오기 보다는 
conmment 에게 memberId가 작성자가 맞는지 물어보는 식의 메서드를 만들어보는건 어떨까요?

객체지향의 `묻지말고 시켜라` 를 검색해보시면 도움이 되실 것 같아요.